doc/changelog/1.21.0-changelog.rst

```diff
@@ -647,7 +647,7 @@ A total of 581 pull requests were merged for this release.
 * `#18851 <https://github.com/numpy/numpy/pull/18851>`__: MAINT: Disable pip version check for azure lint check.
 * `#18853 <https://github.com/numpy/numpy/pull/18853>`__: ENH: Improve the placeholder annotations within sub-modules (part...
 * `#18855 <https://github.com/numpy/numpy/pull/18855>`__: STY: change CRLF line terminators to Unix
-* `#18856 <https://github.com/numpy/numpy/pull/18856>`__: MAINT: Fix the typo "implment"
+* `#18856 <https://github.com/numpy/numpy/pull/18856>`__: MAINT: Fix the typo "implement"
```

Comment:
Another case where a PR title intentionally mentions a typo in its original form, so should probably be left alone

Reply:
Oh!

I'll fix that.  Sorry.

Reply:
fixed